extension-definition-specifications/incident-core/Incident Extension Suite.adoc

```diff
@@ -310,21 +311,23 @@ This is typically used to indicate how an event has affected impacts.
 |[stixtype]#{timestamp_url}[timestamp]#
 |The date and time the event was last recorded. If this is not present it is assumed to be unknown.
 
+If *start_time* and *end_time* properties are both defined, then end_time *MUST* be later than the start_time value. 
```

Comment:
"equal to or later than" or some other variation.  Since we accept the usage of time fidelity it will be common to see the same value for both as many events, tasks, and impacts can be resolved in the same day they they started.

Reply:
Yes - I copied that from somewhere in the STIX spec, but it should be "equal to or later than"